src/providers/object-provider.js

```diff
@@ -330,7 +330,8 @@ export default class YamcsObjectProvider {
         if (defaultAlarm?.staticAlarmRange) {
             return getLimitFromAlarmRange(defaultAlarm.staticAlarmRange);
         } else {
-            throw new Error(`Passed alarm has invalid object syntax for limit conversion`, defaultAlarm);
+            console.warn(`Passed alarm has invalid object syntax for limit conversion`, defaultAlarm);
```

Comment:
I know what this message means in context, but we might make it clearer for other developers if we said something like "Open MCT supports default static alarms only at this time".